api/app/hfi/pdf_generator.py

```diff
@@ -3,16 +3,42 @@
 import pdfkit
 from jinja2 import Environment, FunctionLoader
 from app.schemas.hfi_calc import FireCentre, HFIResultResponse, PlanningArea, WeatherStation
-from app.hfi.daily_template import str_daily_template, CSS_PATH
-from app.hfi.pdf_data_formatter import response_2_daily_jinja_format
+from app.hfi.pdf_template import PDFTemplateName, get_template, CSS_PATH
+from app.hfi.pdf_data_formatter import response_2_daily_jinja_format, response_2_prep_cycle_jinja_format
 
 # Loads template as string from a function
 # See: https://jinja.palletsprojects.com/en/3.0.x/api/?highlight=functionloader#jinja2.FunctionLoader
-jinja_env = Environment(loader=FunctionLoader(str_daily_template), autoescape=True)
+jinja_env = Environment(loader=FunctionLoader(get_template), autoescape=True)
```

Comment:
Does this have to be global scope?

Reply:
No, I kept it in module global scope since it's not mutated and only read from. Do you have a specific way you'd like it scoped? 

I can think of 2 ways:
1. Pass in the `jinja_env` from the called, so this would be scoped to the router function
2. Make these module functions a class and have the env instantiated in an instance

Reply:
I just don't like global scope as it unnecessarily loads on module import. If it loads quickly, I'd rather have it load when needed instead of adding to the memory burden.